docs/DeveloperGuide.md

```diff
@@ -535,20 +560,18 @@ Command and Parser make use of Substitutability:
 * `EditRecipeCommandParser` implements `Parser<EditRecipeCommand>`
 * `EditIngredientCommandParser` implements `Parser<EditIngredientCommand>` <br><br>
 
-The following activity diagram shows how edit operation generally works when a recipe is edited: <br>
+The following two activity diagrams shows how edit operation generally works when a recipe is edited: <br>
```

Comment:
I thought only have one activity diagram?

Reply:
oh i meant to say following two diagrams